_testdata/golint.go

```diff
@@ -0,0 +1,3 @@
+package testdata
+
+var X = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var X should have comment or be unexported